### ISSUE TITLE
Update Homebrew package using bundled Lua libraries

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#660]
After the next release we should be able to update the Homebrew package to us the bundled Lua libraries option instead of requiring the user to invoke `luarocks` manually. I think the Homebrew folks are probably fine with bundling in the sense that we're doing it (which doesn't affect the system or anything else running on it, it just adds batteries to the SILE package).

This will also merit a documentation change to match.

**Comment by alerque** [comment]
This is fixed / no longer relevant. The current Brew formula doesn't use SILE's bundler but it does it itself with the user having to invoke anything, so I think we're golden.

Readme is up to date, the manual needs an overhaul to match, but that's not held up by this issue.
